packages/x-charts/src/RadarChart/RadarAxisHighlight/RadarAxisHighlight.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import composeClasses from '@mui/utils/composeClasses';
+import { RadarAxisSliceHighlight } from './RadarAxisSliceHighlight';
+import { RadarAxisPointsHighlight } from './RadarAxisPointsHighlight';
+import {
+  getRadarAxisHighlightUtilityClass,
+  RadarAxisHighlightClasses,
+} from './radarAxisHighlightClasses';
+
+const useUtilityClasses = (classes: RadarAxisHighlightProps['classes']) => {
+  const slots = {
+    root: ['root'],
+    line: ['line'],
+    slice: ['slice'],
+    dot: ['dot'],
+  };
+
+  return composeClasses(slots, getRadarAxisHighlightUtilityClass, classes);
+};
+
+export interface RadarAxisHighlightProps {
+  /**
+   * Switch between different axis highlight visualization.
+   * - points: display points on each highlighted value. Recommended for radar with multiple series.
+   * - slice: display a slice around the highlighted value. Recommended for radar with a single series.
+   * The default value is computed depending on the number of series provided.
+   */
+  axisHighlightShape: 'points' | 'slice';
```

Comment:
Does it make sense to highlight slices? 🤔 
If yes, should they be less opaque (0.9 maybe)? 😆

Reply:
I don't think it make sense. The different configuration make it look weird. But the point look also weird for single series. That's why I ended up with the current default